SPECS/azure-linux-image-tools/azure-linux-image-tools.spec

```diff
@@ -0,0 +1,102 @@
+%global debug_package %{nil}
+%define our_gopath %{_topdir}/.gopath
+
+Summary:        Azure Linux Image Tools
+Name:           azure-linux-image-tools
+Version:        0.16.0
+Release:        1%{?dist}
+License:        MIT
+URL:            https://github.com/microsoft/azure-linux-image-tools/
+Group:          Applications/System
+Vendor:         Microsoft Corporation
+Distribution:   Azure Linux
+Source0:        https://github.com/microsoft/azure-linux-image-tools/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
+# Below is a manually created tarball, no download link.
+# We're using pre-populated Go modules from this tarball, since network is disabled during build time.
+# We can use the generate-source-tarball.sh script in the given folder along with the package version to build the tarball automatically.
+# In case we need to re-build this file manually:
+#   1. wget https://github.com/microsoft/azure-linux-image-tools/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}.tar.gz -O %%{name}-%%{version}.tar.gz
+#   2. tar -xf %%{name}-%%{version}.tar.gz
+#   3. cd %%{name}-%%{version}
+#   4. go mod tidy
+#   5. go mod vendor
+#   6. tar  --sort=name \
+#           --mtime="2025-07-18 00:00Z" \
+#           --owner=0 --group=0 --numeric-owner \
+#           --pax-option=exthdr.name=%d/PaxHeaders/%f,delete=atime,delete=ctime \
+#           -cf %%{name}-%%{version}-vendor.tar.gz vendor
+#
+Source1:        %{name}-%{version}-vendor-test.tar.gz
+BuildRequires: golang >= 1.24.1
+BuildRequires: systemd-udev
+Requires: imagecustomizer = %{version}-%{release}
+
+%description
+Azure Linux Image Tools. This package provides the Azure Linux Image Customizer tool
+and its dependencies for customizing Azure Linux images.
+
+%package -n imagecustomizer
```

Comment:
Is there a reason to have the main package just install the license and have a second subpackage that is just the binaries?  Consider making the package imagecustomizer or perhaps azlimagecustomizer and drop the -n.  It's not mandatory, but it's not clear why we are splitting these here.

Reply:
Is the idea for the default `azure-linux-image-tools` package to install ALL subpackages like `imagecustomizer` and the future subpackage for the ImageCreator while at the same time the subpackages are to there to allow the user to install a smaller package, if they don't need all of them? If that's the case, it makes sense to me.

I'd only consider keeping the same package prefix to make it clear all of these tools are from the same `azure-linux-image-tools` "umbrella". This boils down to dropping the `-n` switch in `%package` and `%files`.

While on this topic, I'd also consider re-naming "azure-linux" to "azurelinux" to follow the naming we have for other AZL packages like `azurelinux-repos` or `azurelinux-release`.

Reply:
I have updated the package name to `azurelinux-image-tools-*` and remove unnecessary `-n`.